pkgs/applications/window-managers/i3/pystatus.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, libpulseaudio, python3Packages, extraLibs ? [] }:
+
+python3Packages.buildPythonApplication rec {
+  name = "${pname}-${version}";
+  version = "3.34";
+  pname = "i3pystatus";
+  disabled = !python3Packages.isPy3k;
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/i/${pname}/${name}.tar.gz";
+    sha256 = "1bpkkf9q4zqq7fh65zynbv26nq24rfznmw71jjvda7g8kjrwjdk5";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ keyring colour netifaces praw psutil basiciw ] ++
+    [ libpulseaudio ] ++ extraLibs;
+
+  ldWrapperSuffix = "--suffix LD_LIBRARY_PATH : \"${libpulseaudio}/lib\"";
+  makeWrapperArgs = [ ldWrapperSuffix ]; # libpulseaudio.so is loaded manually
+
+  postInstall = ''
+    makeWrapper ${python3Packages.python.interpreter} $out/bin/${pname}-python-interpreter \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
+      ${ldWrapperSuffix}
+  '';
```

Comment:
Can `wrapPythonPrograms` that's in `pythonPackages.wrapPython` be used for this @domenkozar  ?
See https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/generic/wrap.sh